amenity-points.mss

```diff
@@ -316,17 +316,32 @@
     marker-clip: false;
   }
 
-  [feature = 'tourism_information'] {
-    [information != 'guidepost'][zoom >= 17],
+  [feature = 'tourism_information'][zoom >= 17] {
```

Comment:
You need to exclude all the cases that should be rendered later, like `audioguide` or `map`, because otherwise they will be rendered with standard icon from z17 and as a specific icon from z19.

Reply:
Ok I thought that it should be the correct behaviour as they currently appear at z17 with the generic icon. So you prefer that only `office` is displayed at z17 ?